curp/src/server/cmd_worker/mod.rs

```diff
@@ -246,7 +246,7 @@ struct TaskRx<C: Command>(flume::Receiver<Task<C>>);
 
 /// Send cmd to background execution worker
 #[cfg_attr(test, automock)]
-pub(super) trait CEEventTxApi<C: Command + 'static>: Send + Sync + 'static {
+pub(super) trait CEEventTxApi<C: Command + 'static>: Send + Sync + 'static + Debug {
```

Comment:
What is the role of the supertrait `Debug` here?

Reply:
Removed this and manually implement `Debug` for `Context`